index.html

```diff
@@ -119,53 +119,28 @@ <h2>Introduction</h2>
       This specification describes how to secure media types
       expressing Verifiable Credentials and Verifiable Presentations
       as described in the [[VC-DATA-MODEL]], using approaches
-      described by the OAUTH, JOSE, and COSE working groups at IETF. This
-      includes JWTs [[RFC7519]], JWS [[RFC7515]], SD-JWT [[SD-JWT]], and COSE [[RFC9052]],
+      described by the OAUTH, JOSE, and COSE working groups at IETF. 
```

Comment:
`OAuth` is usually written in mixed case, not all caps, as it's not an acronym, including in many if not most pages about the IETF OAuth WG. Barring a strong argument for all caps, I suggest changing this and any other instances of `OAUTH` to mixed case.
```suggestion
      described by the OAuth, JOSE, and COSE working groups at IETF. 
```